spec/requests/request_spec.rb

```diff
@@ -52,14 +52,24 @@
 
     it 'links to an electronic resource with the appropriate display text' do
       get '/catalog/9918309193506421'
-      expect(response.body).to include("<a href=\"/catalog/9918309193506421#view\">Book preliminaries</a>")
+      parsed = Nokogiri::HTML.parse response.body
```

Comment:
Rewriting these tests so that they check the structure of the links, rather than the exact string.  This should make them more resilient against small, non-breaking changes (like reordering attributes, adding new attributes, etc.) and give more manageable diffs (i.e. we don't have to look through the entire response body to see what has changed).